docker/Dockerfile

```diff
@@ -13,8 +13,8 @@ RUN apt-get update && apt-get install -y ffmpeg libsm6 libxext6 git ninja-build
     && rm -rf /var/lib/apt/lists/*
 
 # Install MMCV
-RUN pip install --no-cache-dir --upgrade pip wheel setuptools
-RUN pip install --no-cache-dir mmcv-full==1.3.17 -f https://download.openmmlab.com/mmcv/dist/cu101/torch1.6.0/index.html
+RUN ["/bin/bash", "-c", "pip install --no-cache-dir --upgrade pip wheel setuptools"]
```

Comment:
This will result in smaller image size. Pip caches do not need to be kept in Docker images. [https://stackoverflow.com/questions/45594707/what-is-pips-no-cache-dir-good-for](this problem on StackOverflow)